lib/tokenable.rb

```diff
@@ -0,0 +1,11 @@
+module Tokenable 
```

Comment:
This file could as well live inside `models/concerns/` directory and you would not have to add lib to autoloaded paths. But you can leave it as it is too.